internal/tokenizer/tokenize.go

```diff
@@ -8,6 +11,9 @@ import (
 
 const byteLimit = 100000
 
+// Tokenize returns classification tokens from content. The tokens returned
```

Comment:
I would suggest remove "classification" from the description of this method, as this API is basically exposes only a language-agnostic scanner/lexer that could be used for anything.